config/zones.json

```diff
@@ -1477,6 +1477,16 @@
   "US-HI": {
     "flag_file_name": "us.png"
   },
+  "US-NY": {
+    "parsers": {
+      "production": "US_NY.fetch_production"
+    },
+    "flag_file_name": "us.png",
+    "contributors": [
+      "https://github.com/corradio",
```

Comment:
No need to add my name :)

Reply:
Sure?  You split the US on the map.

Reply:
yeah let's keep people who worked exclusively on the parser